packages/react/src/FronteggProvider.tsx

```diff
@@ -47,6 +47,14 @@ export const Connector: FC<ConnectorProps> = ({ history, appName, isExternalHist
     if (opts?.preserveQueryParams || isAuthRouteRef.current(path)) {
       path = `${path}${window.location.search}`;
     }
+
+    const stepUpUrl = props.authOptions?.routes?.stepUpUrl;
```

Comment:
add options `byPassFrameworkRouter` and use it from the SDK
Another option is to create list of `routesToByPassReactRouter` and then
```
if(routesToByPassReactRouter.some(route=> path.startsWith(route))){
      window?.history?.pushState(null, '', path);
      return;
}
```

Reply:
fixed